clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -4509,6 +4509,19 @@ llvm::Constant *CodeGenModule::GetOrCreateMultiVersionResolver(GlobalDecl GD) {
   return Resolver;
 }
 
+bool CodeGenModule::shouldDropDLLAttribute(const Decl *D,
```

Comment:
This doesn't access any members of CodeGenModule, so making it a static non-member function would be better.

Reply:
Never mind, shouldMapVisibilityToDLLExport() is a member, so this makes sense.